### ISSUE TITLE
npm install fails

### ISSUE THREAD

**Issue by Tyrdall** [mbrochh/django-reactjs-boilerplate#1]
This happens because of a dependency conflict.

```
npm ERR! peerinvalid The package eslint@2.7.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer babel-eslint@5.0.4 wants eslint@<2.3.0
```

I had to run `npm install eslint@2.2.0` afterwards to fix it.

**Comment by samorajp** [comment]
same solution by me.

**Comment by vinamelody** [comment]
Info: this could happen on step 3
How about bumping the version `"babel-eslint": "^6.0.2",` ? it worked when i tried.

**Comment by mbrochh** [comment]
@vinamelody yes this is obviously the solution but babel 6 is drastically different from 5 and breaks everything. i have tried two times already to get everything working, but failed. will try a third time, soon ;)

**Comment by arrchana** [comment]
I too had to run `npm install eslint@2.2.0` to fix it.
